config/routes.rb

```diff
@@ -9,6 +9,11 @@
   get "/users/:id", to: "users#show", as: "user"
   
   get "/users/:id/posts", to: "posts#index", as: "user_posts"
-  get "/users/:id/posts/:id", to: "posts#show", as: "user_post"
-
+  get "/posts/new", to: "posts#new", as: "user_posts_new"
+  post "/posts", to: "posts#create", as: "user_posts_create"
+  get "/posts/:id/new", to: "comments#new", as: "user_post_comments_new"
+  post "/posts/:id", to: "comments#create", as: "user_post_comments_create"
+  post "/posts/:id/like", to: "likes#create", as: "user_post_likes"
+  get "/posts/:id", to: "posts#show", as: "user_post"
```

Comment:
- [ ]  Great work implementing the changes requested, however, am getting the routes error below whenever I try  to access the forms to create a post 

![image](https://github.com/biruk-bereka/Blogify/assets/61408860/45263fba-edc4-430c-a0e6-be910237bcad)